db/migrate/20220102000108_delete_user_profile_address.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteUserProfileAddress < ActiveRecord::Migration[6.1]
```

Comment:
I think this migration can be removed together with next one for adding this field back